package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public --no-clear"
+    "start": "sirv public --single --no-clear"
   },
```

Comment:
@lsh58 `--single` 옵션을 무슨 기능인지 궁금합니다!!

Reply:
When you use the History API and point the browser to the root path / (usually /index.html) all links and Routes will work properly. But when you start your app on any subpage, like /page/subpage, you will see the 404 Not found error. Because of this, you need to setup your server to point all requests to /index.html.

--single을 넣지 않으면 root page외의 페이지로 열때 404error가 나온다고 합니다. (왜인지까지는 안나와있네요;)
실제로 해보면, --single을 안넣었을때는 a태그등으로 이동은 가능하지만,  url입력으로 이동이 불가능합니다.

Reply:
@lsh58 넵, 확인 감사합니다!
아마 `historyApiFallback` 관련 설정인가보네요 ~!
아마 index.html 파일을 서빙할텐데, /index 외의 path로 진입하면 html이 없어서 404일거고,
그래서 /index 이외의 path로 진입할 때, /index의 svelte 앱을 불러오고 라우팅을 매핑해주는 방식일 겁니다!